src/Symfony/Component/Messenger/Bridge/AmazonSqs/Tests/Transport/AmazonSqsQueryStringParserTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\AmazonSqs\Tests\Transport;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Bridge\AmazonSqs\Transport\AmazonSqsQueryStringParser;
+
+class AmazonSqsQueryStringParserTest extends TestCase
+{
+    public function testParse()
+    {
+        $queryStrings = 'access_key=%[\^$.|?*+-*${}/?!"%()&secret_key=A+a+B+b&region=%24&queue_name=';
```

Comment:
This will break the test:
```suggestion
        $queryStrings = 'access_key=%[\^$.|?*+-*${}/?!"%()+&secret_key=A+a+B+b&region=%24&queue_name=';
```
---
Please replace only `+` by a space withing the `secret_key`